packages/authentication/test/acceptance/basic-auth.ts

```diff
@@ -0,0 +1,206 @@
+// Copyright IBM Corp. 2013,2017. All Rights Reserved.
+// Node module: loopback
+// This file is licensed under the MIT License.
+// License text available at https://opensource.org/licenses/MIT
+
+import {
+  Application,
+  Server,
+  api,
+  OpenApiSpec,
+  ParameterObject,
+  ServerRequest,
+  ServerResponse,
+  parseOperationArgs,
+  writeResultToResponse,
+  ParsedRequest,
+  OperationArgs,
+  FindRoute,
+  InvokeMethod,
+  getFromContext,
+  bindElement,
```

Comment:
Can these default actions be grouped into a namespace or `Sequence`? Having to import such a big list from `core` is a warning sign.